backend/src/test/java/botobo/core/utils/Fixture.java

```diff
@@ -1,84 +1,171 @@
+                    Map.of(4L, "스프링"),
+                    Map.of(5L, "네트워크")
+            )
+    );
+
+    public static final List<WorkbookRequest> WORKBOOK_REQUESTS_WITH_TAG =
+            문제집명과_태그_요청이_포함된_문제집_요청_만들기(
+                    List.of(
+                            Map.of("Java", true),
+                            Map.of("JAVAVA", true),
+                            Map.of("Javascript", true)
+                    ),
+                    List.of(TAG_REQUESTS_TYPE_1, TAG_REQUESTS_TYPE_2, TAG_REQUESTS_TYPE_3)
+            );
+
+    /**
+     * 주의, namedAndOpenedSets와 tagRequests의 사이즈는 동일해야하며
+     * 병렬적으로 매핑되어 WorkbookRequest를 생성합니다.
+     *
+     * @param nameAndOpenedSets
+     * @param tagRequests
+     * @return 태그가 포함된 WorkbookRequest List
+     */
+    private static List<WorkbookRequest> 문제집명과_태그_요청이_포함된_문제집_요청_만들기(List<Map<String, Boolean>> nameAndOpenedSets,
+                                                                     List<List<TagRequest>> tagRequests) {
+
+        if (nameAndOpenedSets.size() != tagRequests.size()) {
+            throw new IllegalArgumentException("문제집명과 태그 요청의 수는 같아야합니다.");
+        }
+        List<WorkbookRequest> requests = new ArrayList<>();
+        int size = nameAndOpenedSets.size();
+
+        for (int i = 0; i < size; i++) {
+            Map<String, Boolean> nameAndOpened = nameAndOpenedSets.get(i);
+            List<TagRequest> tagRequest = tagRequests.get(i);
+            for (Map.Entry<String, Boolean> entry : nameAndOpened.entrySet()) {
+                requests.add(
+                        WorkbookRequest.builder()
+                                .name(entry.getKey())
+                                .opened(entry.getValue())
+                                .tags(tagRequest)
+                                .build()
+                );
+            }
+        }
+        return requests;
+    }
+
+    private static List<TagRequest> 태그명과_아이디가_포함된_태그_요청_만들기(List<Map<Long, String>> idAndNameSets) {
+        List<TagRequest> requests = new ArrayList<>();
+
+        for (Map<Long, String> args : idAndNameSets) {
+            for (Map.Entry<Long, String> entry : args.entrySet()) {
+                requests.add(
+                        TagRequest.builder()
+                                .id(entry.getKey())
+                                .name(entry.getValue())
+                                .build()
+                );
+            }
+        }
+        return requests;
+    }
+
+    public static final List<CardRequest> CARD_REQUESTS_OF_30_CARDS =
+            질문_답변_문제집_아이디가_포함된_카드_요청_만들기(
+                    질문_답변_문제집_아이디를_띄어쓰기로_구분해_30개_생성()
+            );
+
+    private static List<String> 질문_답변_문제집_아이디를_띄어쓰기로_구분해_30개_생성() {
+        List<String> results = new ArrayList<>();
+        for (int i = 1; i <= 30; i++) {
+            long id = ((i - 1) / 10) + 1;
+            results.add(i + " answer " + id);
+        }
+        return results;
+    }
```

Comment:
이 부분은 CardRequest 에 필요한 인자가 3개라서 String 으로 만들고 공백으로 분리하셨군요!
클래스를 만들어서 사용한다면 가독성이 좋아지지 않을까 셍각해요 👍  
글구 사소한 부분인데 그냥 숫자를 넣는 것보다 조금의 정보를 더 넣어주면 좋을 것 같아요.

```suggestion
    private static List<CardRequestInfo> 질문_답변_문제집_아이디를_띄어쓰기로_구분해_30개_생성() {
        List<CardRequestInfo> cardRequestInfos = new ArrayList<>();
        for (int i = 1; i <= 30; i++) {
            CardRequestInfo cardRequestHint = new CardRequestInfo();
            cardRequestHint.question = "question" + i;
            cardRequestHint.answer = "answer" + i;
            cardRequestHint.workbookId = ((i - 1) / 10) + 1L;
            cardRequestInfos.add(cardRequestHint);
        }
        return cardRequestInfos;
    }

    private static class CardRequestInfo {
        String question, answer;
        Long workbookId;
    }
```

Reply:
좋아효!!!!!! 이부분도 B16에서 꼭 반영할게요 👍